libs/blocks/quiz-entry/quiz-entry.css

```diff
@@ -1,3 +1,8 @@
+.quiz-entry {
+  --quiz-button-disabled-bg: #757575;
+  --quiz-button-disabled-text: #FFF6F6;
+}
```

Comment:
I don't think we need to declare variables as they're not used in multiple places

Reply:
The point here isn't so much that it would be used in more than one place as it is to make it easier when they are updated. An update to these colors is imminent pending a consonant review - and these may very well be used in multiple places depending on what they decide to do.